oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/LoginEndpoint.java

```diff
@@ -17,6 +20,7 @@
 public class LoginEndpoint {
   private String username;
   private String password;
+  private String errormessage;
```

Comment:
use camelCase when naming - errorMessage

Reply:
It it not camelCase because the LoginEndpoint receives it as a get parameter. If I change it to errorMessage then the GET parameter must also be camelCase.